ompi/runtime/ompi_mpi_params.c

```diff
@@ -3,7 +3,7 @@
  * Copyright (c) 2004-2005 The Trustees of Indiana University and Indiana
```

Comment:
I think I would move all the changes in this file in the ompi_spc.c and create a registration function that is called from ompi_mpi_params.c.